INSTALL.md

```diff
@@ -77,3 +77,16 @@ Please see [INSTALL.make.md](dev/doc/INSTALL.make.md) for build and
 installation instructions using `make`. If you wish to experiment with
 the Dune-based system see the [dune guide for
 developers](dev/doc/build-system.dune.md).
+
+Notes on build-time dependencies
+--------------------------------
+
+- The OCaml compiler and findlib are build-time dependencies, but also
+  run-time dependencies if you wish to use the native compiler.
+
+- When loading plugins, you should make sure that these were compiled
+  with the same OCaml setup (version, flags, dependencies...) as Coq.
+  This is also true when loading `vo` files.  For this reason, it is
+  rarely a good idea to distribute pre-compiled `vo` files to the
+  general public (if you don't know what software they will use to
+  load them).
```

Comment:
This makes it sound like a bit like they might be using something other than Coq to load vo files.  I'd suggest
```suggestion
  general public (if you don't know whether they'll have the same
  OCaml and Coq setup as was used for compilation).
```